ocs_samples/basic_samples/SDS/DotNet/SdsRestApiCore/SdsRestApiCore/Program.cs

```diff
@@ -617,19 +616,17 @@ public static async Task<bool> MainAsync(bool test = false)
                 if(waveStreamSecond.Indexes !=null )
```

Comment:
this if statement and the one above on line 613 should have curly braces around the code block even though it's just one line to keep the style consistent throughout the project